src/store/getters/state-getters.ts

```diff
@@ -100,8 +100,7 @@ export const isGovAccountUser = (state: StateIF): boolean => {
 
 /** Whether the user is SBC Staff (which is not the same as Staff). */
 export const isSbcStaff = (state: StateIF): boolean => {
-  const orgInfo = getOrgInformation(state)
-  return (orgInfo?.orgType === 'SBC_STAFF')
+  return state.stateModel.accountInformation?.accountType === AccountTypes.SBC_STAFF
```

Comment:
You might as well use `getAccountInformation` getter here instead of dereferencing the state.

Reply:
And maybe move this getter near `isPremiumAccount` in this file to keep account getters together.

Reply:
sure